3drepo/3drepo.io
# ISSUE #4242 no dots in property/module names

This fixes #4242

#### Description
- `.` should no longer be allowed in property names and module names

#### Test cases
- user should not be able to 
  - create a template with property/module name with `.` in it
  - update a template to have a property/module name with `.` in it
